src/main/java/com/puppycrawl/tools/checkstyle/checks/FinalParametersCheck.java

```diff
@@ -86,6 +110,24 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
+ * Example :
+ * </p>
+ * <pre>
+ * class MyClass {
+ *   public MyClass(final int num) { // OK
+ *   }
+ *   public void myMethod(String str) { // violation, str should be final
+ *   }
+ * }
+ *
+ * class MyClass {
+ *   public MyClass(final int num) { // OK
```

Comment:
please combine this class with previous.

there should be single class in example HTML tag and the same class used for different checkstyle configs, to let user clearly see how configuration changes change behavior of the same code.